src/components/molecules/Darkmode/Darkmode.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Icon } from 'atoms'
+
+const Darkmode = ({ isActive, toggle, cssStyles, isSupported }) => {
+  if(!isSupported) {
+    return null
+  }
+
+  return (
+    <React.Fragment>
+      <Icon className='nav-darkmode fas fa-adjust' color='white' size='md' onClick={() => toggle(!isActive)} />
```

Comment:
@Joe-Pena if we change the toggle handler per my suggestion below, then the onClick would just be `onClick={toggleActive }`